olm/olm.sh

```diff
@@ -112,30 +145,20 @@ installOperatorMarketPlace() {
       echo "Catalog source not created after 2 minutes"
       exit 1
     fi
-
-    kubectl get catalogsource/"${packageName}" -n "${marketplaceNamespace}" -o json | jq '.metadata.namespace = "olm" | del(.metadata.creationTimestamp) | del(.metadata.uid) | del(.metadata.resourceVersion) | del(.metadata.generation) | del(.metadata.selfLink) | del(.status)' | kubectl apply -f -
-    marketplaceNamespace="olm"
   fi
 
+  echo "Subscribing to version: ${CSV}"
+
   kubectl apply -f - <<EOF
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: ${namespace}
----
-apiVersion: operators.coreos.com/v1alpha2
+apiVersion: operators.coreos.com/v1
```

Comment:
Is this change from `v1alpha2` to `v1` OK for any platform ? I assume it is for the last Minikube, but what about OpenShift 4.2 or other platforms ? Are there constraints that require us changing this in this PR ?

Reply:
In fact I had problem in Openshift 4.3.1 with this version. With **v1** I don't have any problem in minikube or Openshift 4.x

Reply:
OK